WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -163,7 +163,21 @@ private async Task OnExportPsbtAsync()
 	{
 		if (Transaction is { })
 		{
-			var saved = await TransactionHelpers.ExportTransactionToBinaryAsync(Transaction);
+			bool saved = false;
+			try
+			{
+				saved = await TransactionHelpers.ExportTransactionToBinaryAsync(Transaction);
+			}
+			catch (UnauthorizedAccessException uex)
+			{
+				Logger.LogError(uex);
+				await ShowErrorAsync("Transaction Building", uex.ToUserFriendlyString(), "Wasabi was unable to save the PSBT.");
+			}
```

Comment:
You don't have to separately handle this here. 
Rather handle it here: https://github.com/zkSNACKs/WalletWasabi/blob/4c9f288e17ed5f57698ba6c5c0334bf8d841e5a7/WalletWasabi.Fluent/Extensions/FriendlyExceptionMessageExtensions.cs#L12